### PR TITLE
[WIP] Moving parametrize structure with minimal global fixture

### PR DESCRIPTION
**Pull request by Yadnyawalkya**
{{pytest: cfme/tests/v2v/test_post_migrations.py -k test_migrations_vm_attributes --use-provider rhv42 --use-provider vsphere65-nested --provider-limit 2 -vvvv }}

This is just proposal, going to move other tests with same pattern form v2v directory.

### CODE REVIEW

**Comment by Yadnyawalkya** [comment]
@sshveta  @digitronik  need thoughts?

**Comment by Yadnyawalkya** [comment]
Yes, agree, @sshveta  go ahead with changes, not putting this to RFR till your PR get merge as discussed.
Just wanted to take suggestions if this is ideal solution. Adding it in new structure.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by Yadnyawalkya** [comment]
So... @sshveta's PR is merged and my changes works over her changes (PRT passed). I am adding more files to follow this proposal.

File: cfme/fixtures/v2v.py [hunk 1]
```diff
@@ -191,6 +193,35 @@ def _form_data(source_provider, provider):
...
+    vm_obj = collection.instantiate(
+        vm_name, source_provider, template_name=rhel7_minimal(source_provider)["name"]
+    )
+    vm_obj.create_on_provider(
+        timeout=2400,
+        find_in_cfme=True,
+        allow_skip="default",
+        datastore=source_datastore,
+        power_on=True
+    )
```
Comment (sshveta) [inline cfme/fixtures/v2v.py]: Source datastore and creating vm can be moved to a fixture .



    def create_vm_on_provider():
         source_datastores_list = source_provider.data.get("datastores", [])
         source_datastore = [d.name for d in source_datastores_list if d.type == "nfs"][0]
        collection = source_provider.appliance.provider_based_collection(source_provider)
        vm_name = random_vm_name("v2v-auto")
        vm_obj = collection.instantiate(
        vm_name, source_provider, template_name=rhel7_minimal(source_provider)["name"])
        vm_obj.create_on_provider(
              timeout=2400,
              find_in_cfme=True,
              allow_skip="default",
              datastore=source_datastore,
              power_on=True
        )
    yield vm
    vm_obj.cleanup_on_provider()
    infrastructure_mapping_collection.delete(mapping)

**Pull request closed without merging**